client/src/components/ModelGraphViewerComponent/ModelGraphViewerCytoscapeComponent/ModelGraphViewerCytoscapeComponent.js

```diff
@@ -216,6 +211,24 @@ export class ModelGraphViewerCytoscapeComponent extends React.Component {
     });
   }
 
+  sessionLayout() {
```

Comment:
Can you help me understand how this function works? It looks like that it saves the layout if it doesn't exist but applies the layout if it does exist?

Perhaps it would be clearer if there were separate methods for each of these? So they could be called explicitly?

Reply:
The Idea is to Store the Layout values the first time a  new Layout is loaded and retrieve the previous states if the  layout was calculated in any previous moment, this is because the cytoscape component will retrieve different layouts even if you don't add or move the nodes (they are calculated by the library and retrieve to the model) so if a node is moved and I store the new position instead of the whole thing when the layout is changed back the rest of the nodes will have different positions. 
Doing it this way I ensure that it does not mater how many nodes are moved or stayed untouched the value from the session would be exactly the one expected .

I did change it into two new methods to make it more explicit

Reply:
@jamescarpinter - ok with this response?